src/Framework/Analyzer/src/DiagnosticDescriptors.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.AspNetCore.Analyzers;
+
+[System.Diagnostics.CodeAnalysis.SuppressMessage("MicrosoftCodeAnalysisReleaseTracking", "RS2008:Enable analyzer release tracking")]
+internal static class DiagnosticDescriptors
+{
+    internal static readonly DiagnosticDescriptor DoNotUseModelBindingAttributesOnMinimalActionParameters = new(
+        "ASP0003",
+        "Do not use model binding attributes with Map actions",
+        "{0} should not be specified for a {1} delegate parameter",
+        "Usage",
+        DiagnosticSeverity.Warning,
+        isEnabledByDefault: true,
+        helpLinkUri: "https://aka.ms/aspnet/analyzers");
+
+    internal static readonly DiagnosticDescriptor DoNotUseConfigureWebHostWithConfigureHostBuilder = new(
+        "ASP0005",
```

Comment:
Where did 4 go... 🤔

Maybe I can't count?

Reply:
`ASP0004` should map to `DoNotReturnActionResultsFromMapActions`.  I think you might have missed a commit during a rebase or something.

`ASP0005` is also taken in a recent PR that I've merged: https://github.com/dotnet/aspnetcore/blob/d9ac5f4041e7912cfec08656a775b4418f203d76/src/Framework/Analyzer/src/DelegateEndpoints/DiagnosticDescriptors.cs#L29

Unfortunately, there's gonna be a lot of stomping over this source file as we build out analyzers in the space.